src/main/java/amidst/mojangapi/minecraftinterface/MinecraftInterface.java

```diff
@@ -59,7 +62,23 @@
 				int x, int y, int width, int height,
 				boolean useQuarterResolution, Function<int[], T> biomeDataMapper)
 				throws MinecraftInterfaceException;
-
+	}
+	
+	/**
+	 * An interface that includes methods for retrieving all of the configuration options
+	 * needed for creation of a world that don't require things like a seed, world type,
+	 * or generation options.
+	 * 
+	 * This should be able to access methods supplied in the SymbolicClasses just like
+	 * WorldAccessor can.
+	 * 
+	 * This also includes the createWorldAccessor method so it's required to create
+	 * a WorldConfig first.
+	 */
+	public static interface WorldConfig {
+		
 		public Set<Dimension> supportedDimensions();
+		
+		public WorldAccessor createWorldAccessor(long seed, WorldType worldType, String generatorOptions) throws MinecraftInterfaceException;
```

Comment:
I'm not sure about this `WorldConfig`/`WorldAccessor` split; I don't really see the point of the `WorldConfig` objects, given that in all `MinecraftInterface`s implementations these objects are basically empty.

The only benefit is that the initialisation code can be run separately, but why can't this be bundled with the creation of the first world?

Reply:
I removed all of the `WorldConfig` stuff and instead changed some stuff in `ThreadedWorldAccessor` as well as `LoggingMinecraftInterface` so it works like it used to while still having multithreading.